packages/atlas-meta-server/src/tags.ts

```diff
@@ -3,7 +3,8 @@ import { formatISODuration } from 'date-fns'
 import { BasicChannelFieldsFragment, BasicVideoFieldsFragment } from './api/__generated__/sdk'
 import { joinUrlFragments } from './utils'
 
-const BASE_ATLAS_URL = 'https://play.joystream.org'
+const BASE_APP_URL = 'https://play.joystream.org'
```

Comment:
If we're going through this exercise let's make this configurable via env as well

Reply:
@drillprop This wasn't resolved